src/slices/favorites/favoritesActions.ts

```diff
@@ -0,0 +1,61 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { FavoritesState } from "./favoritesSlice";
+import { RootState } from "../../../store";
+import { writeToLog } from "../../helpers/LogHelper";
+
+export const loadFavorites = createAsyncThunk(
+  "Favorites/loadFavorites",
```

Comment:
Just to maintain continuity, probably should use a lowercase f. Same for the AsyncStorage key.

Reply:
Ok, I'll change it to lowercase f. 
I was referencing settingsActions.ts and it is using a capital S, but I see that others are using lowercase.